hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright YEAR The Kubernetes Authors.
```

Comment:
Should this say Kubernetes or Dynatrace?

Reply:
Ah thanks - changed!